SLC-S-GQIMonitor/SLC-S-GQIMonitor/CatalogInformation/README.md

```diff
@@ -10,5 +10,10 @@ Currently, the app offers 4 distinct pages.
 - Metric history: overview of all metrics available on the system
 - Application metrics: overview of the top 10 applications for each metric
 - User metrics: overview of the top 10 users for each metric
+- Extension logs: search and inspect log entries for each extension library
+
+
+Prerequisites
```

Comment:
1. GQI DxM will also be a prerequisite in order to see the logs.
2. Maybe we should make clear that 10.5.9 + DxM is only required for the logs.
Other functionality should still work regardless